src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public CinemaHall get(Long id) {
+        return cinemaHallDao.get(id).get();
```

Comment:
I dont think that it's ok use get, what happens if in dao get will nothing find?
```suggestion
        return cinemaHallDao.get(id).orElseThrow(() ->
                new RuntimeException("Can't get cinema hall by id: " + id));
```

Reply:
If so, I get NoSuchElementException and the path to this line, don't see any problems here